jvcproj/plugin.yaml

```diff
@@ -19,7 +19,20 @@ plugin:
 
 parameters:
     # Definition of parameters to be configured in etc/plugin.yaml
-    
+    host:
+        type: ip
+        default: 0.0.0.0
+        description:
+            de: 'Die IP Adresse des JVC Projektors ist anzugeben.'
+            en: 'Define IP address of your JVC projector.'
+
+    gammaconf_dir:
+        type: dict
```

Comment:
This should be type `str` because you configured a path, not a dictionary directly.